mapillary_tools/process_video.py

```diff
@@ -49,30 +52,77 @@ def sample_video(video_file,
 
     # command specific checks
     video_file = os.path.abspath(video_file) if video_file else None
-    if video_file and not os.path.isfile(video_file):
-        print("Error, video file " + video_file +
-              " does not exist, exiting...")
+    if not video_file:
```

Comment:
same comment as above, was this check simplified just because video_file can be a dir of videos?